pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -234,8 +236,12 @@ func CreateGCECloud(projectID, region, zone string, managedZones []string, netwo
 	}, nil
 }
 
-// Initialize passes a Kubernetes clientBuilder interface to the cloud provider
-func (gce *GCECloud) Initialize(clientBuilder controller.ControllerClientBuilder) {}
+// Initialize takes in a clientBuilder and spawns a goroutine for watching the clusterid configmap.
+// This must be called before utilizing the funcs of gce.ClusterId
+func (gce *GCECloud) Initialize(clientBuilder controller.ControllerClientBuilder) {
```

Comment:
`kube-controller-manager` & `cloud-controller-manager` do.

The GCE cloud provider is hitting some growing pains with regard to the rigid *cloud* interface. With the eventual removal of cloud providers from the kubernetes repo (and cloud-dependent controllers), `Initialize` should become defunct in the long run.